README.md

```diff
@@ -86,7 +86,7 @@ Documenso is built using awesome open source tech including:
 - [Node SignPDF (Digital Signature)](https://github.com/vbuch/node-signpdf)
 - [React-PDF for viewing PDFs](https://github.com/wojtekmaj/react-pdf)
 - [PDF-Lib for PDF manipulation](https://github.com/Hopding/pdf-lib)
-- Check out /packages.json and /apps/web/package.json for more
+- Check out `/packages.json` and `/apps/web/package.json` for more
```

Comment:
If we're fixing, this should be `package.json` no?

Reply:
yep.